src/modules/blockchainApplication/explore/mocks/index.js

```diff
@@ -0,0 +1,28 @@
+import { rest } from 'msw';
+import { API_VERSION, LIMIT } from 'src/const/config';
+import { mockBlockchainApp, mockBlockchainAppStatistics } from '@blockchainApplication/explore/__fixtures__';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Since you have exported two modules, I don't think you need to disable this rule.